manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "b2b-orders-history",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

We can use a beta version, just to avoid of burning this version

```suggestion
  "version": "0.3.0-beta.0",
```